test/e2e/wmco_test.go

```diff
@@ -49,15 +49,15 @@ func TestWMCO(t *testing.T) {
 	// currently wait for nodes to fully reconcile after changing the private key back to the valid key. Any tests
 	// added/moved in between these two suites may fail.
 	// This limitation will be removed with https://issues.redhat.com/browse/WINC-655
-	t.Run("reconfigure", reconfigurationTestSuite)
-	t.Run("destroy", deletionTestSuite)
+	t.Run("reconfigure", tc.reconfigurationTestSuite)
+	t.Run("destroy", tc.deletionTestSuite)
```

Comment:
Any reason for only doing this to the reconfigure and deletion suites? IMO we should be consistent in either creating a new test context for each suite or reusing the top level test context across all suites (creation, network, upgrade, storage etc)